src/app/zones/views/ZoneEgressListView.vue

```diff
@@ -13,125 +13,132 @@
         },
       ]"
     >
-      <div class="zoneegresses">
-        <div class="stack">
-          <div class="kcard-border">
-            <DataOverview
-              :selected-entity-name="entity?.name"
-              :page-size="PAGE_SIZE_DEFAULT"
-              :is-loading="isLoading"
+      <template #title>
+        <h2>
+          <RouteTitle
+            :title="t('zone-egresses.routes.items.title')"
+            :render="true"
+          />
+        </h2>
+      </template>
+
+      <DataSource
+        v-slot="{ data, error }: ZoneEgressOverviewCollectionSource"
+        :src="`/zone-egresses?size=${props.size}&page=${props.page}`"
+      >
+        <KCard>
+          <template #body>
+            <AppCollection
+              class="zone-egress-collection"
+              data-testid="zone-egress-collection"
+              :headers="[
+                { label: 'Name', key: 'name' },
+                { label: 'Status', key: 'status' },
+                { label: 'Actions', key: 'actions', hideLabel: true },
+              ]"
+              :page-number="props.page"
+              :page-size="props.size"
+              :total="data?.total"
+              :items="data ? transformToTableData(data.items) : []"
```

Comment:
If data isn't set yet it means `:items` should be empty as in "undefined empty", rather than an empty array, which would translate to "this has loaded but there are zero items". TLDR; having `[]` as the default here breaks the loading skeleton.

Reply:
Fixed in https://github.com/kumahq/kuma-gui/pull/1209/commits/d04c1e36a7a72697a79010fc619f3e6c9bb5841b